app/services/animal_service.py

```diff
@@ -127,7 +127,7 @@ def insert_form(self, animal_id: str, form_id: str, request_id: str = "") -> Res
                 )
                 if result:
                     self.logger.info(f"id={request_id} Form inserted")
-                    return ResponseDTO(result, "Form inserted", http.HTTPStatus.OK)
+                    return ResponseDTO({"result": result}, "Form inserted", http.HTTPStatus.OK)
```

Comment:
Aqui basta ser result
```suggestion
                    return ResponseDTO(result, "Form inserted", http.HTTPStatus.OK)
```

Reply:
vdd.. tava dando erro  na serialização, achei que podia ser ai, mas aparentemente não é :v